src/commcare_cloud/commands/terraform/terraform_migrate_state.py

```diff
@@ -0,0 +1,256 @@
+from __future__ import print_function
+import json
+import sys
+import tempfile
+from copy import deepcopy
+
+import jsonobject
+import os
+import re
+import runpy
+import subprocess
+from collections import namedtuple, deque
+
+from commcare_cloud.alias import commcare_cloud
+from commcare_cloud.cli_utils import ask, print_command
+from commcare_cloud.commands.command_base import CommandBase, CommandError
+from commcare_cloud.environment.main import get_environment
+
+
+class TerraformMigrateState(CommandBase):
+    command = 'terraform-migrate-state'
+    help = "Apply unapplied state migrations in commcare_cloud/commands/terraform/migrations"
```

Comment:
@snopoke in terms of docs, I'm thinking just a better (and much longer) help message here would probably be the place to put it. Is that along the lines of what you were thinking?

Reply:
That's fine though I think we could do with some more general AWS docs to explain the overall setup.